library/core/src/ptr/const_ptr.rs

```diff
@@ -1920,6 +1964,279 @@ mod verify {
     use crate::kani;
     use core::mem;
     use kani::PointerGenerator;
+    // Constant for array size used in all tests
+    const ARRAY_SIZE: usize = 5;
```

Comment:
Could you add a proper justification for `5`? How this "bound" impacts the proofs? What is the verification performance for this number? Are we missing anything by staying within this bound?

Reply:
@szlee118 FYI

Reply:
There is an [explanation](https://github.com/model-checking/verify-rust-std/pull/166#discussion_r1861482793) in previous comments. Just to confirm if this justification is required to be comment in the code? We are currently gathering verfication details in final documents.

Reply:
See https://github.com/model-checking/verify-rust-std/pull/166#discussion_r1864993769 for my 2c on this.

Reply:
Will this comment be address by another PR?

Reply:
Yes it will be addressed by another PR.